Formula/lanraragi.rb

```diff
@@ -33,8 +33,8 @@ class Lanraragi < Formula
 
   uses_from_macos "libarchive"
 
-  on_macos do
-    resource "libarchive-headers" do
+  resource "libarchive-headers" do
```

Comment:
Could probably be done in a separate PR, but it seems much more maintainable to use Homebrew `libarchive` than to workaround any missing headers by downloading them.